documentation/docs/getting-started/verifiable_credentials/overview.md

```diff
@@ -9,7 +9,38 @@ keywords:
 - person
 ---
 
+Credentials are statements about an entity, such as properties that the entity posseses or capabilities that they have (like drivers licences, passports, or a person's age). Verifiable Credentials (VCs) are statements about the holder of the credential that can be cryptographically verified by a third party, either online or in person. Additionally, the holder decides what is shared and who it is shared.
 
-Verifiable Credentials are statements about the holder. They can be verified online or in person, and the holder decides who to share them with.
+There are several types of actors that play different roles in a verifiable credential system. We'll start with a common example from the world today and outline the roles that various entities play.
 
-[Verifiable Credentials Data Model 1.0 Specification](https://w3c.github.io/vc-data-model/)
\ No newline at end of file
+:::tip Example - Passport Issuance
+
+A government (the _Issuer_) issues a passport asserting citizenship (the _Verifiable Credential_) to Alice (the _Subject_ and _Holder_), and writes the information to a database (the _Verifiable Data Registry_). When crossing the border, Alice (the _Holder_) presents her passport to a border agent (the _Verifier_) who can verify that Alice (the _Subject_) is indeed a citizen.
```

Comment:
> and writes the information to a database (the _Verifiable Data Registry_)

We don't do this. The government signs the VC with a public key, stored on a Tangle. The only copy of the passport that needs to exist is in Alice's control (in an App or something).

Reply:
Ah sorry, is this text representing the current physical / digital centralized format?

Reply:
Yes sorry, my plan here was to illustrate a real world example (which as you mention is currently done in physical / centralized format) and use that to explain the parts of the VC spec and how IOTA implements them.

I was thinking more generally that having a consistent end-to-end narrative throughout that ties all of the IOTA client libraries would help folks reading it to build up a proper mental model.

Thoughts on the best way to clarify this? I could re-word the last sentence of the paragraph before to explicitly call out that this is "an example of how things work in the world today" (or something like that). wdyt?

Reply:
Yea, the explicit call-out would work :)

Reply:
The sentence before the example now reads:

"We'll start with a common example of how things work in the world today using physical credentials and centralized databases, and outline the roles that various entities play in the Verifiable Credential system.""